javascript/stimulus_reflex.js

```diff
@@ -103,7 +108,32 @@ if (!document.stimulusReflexInitialized) {
   document.stimulusReflexInitialized = true;
   window.addEventListener('load', setup);
   document.addEventListener('turbolinks:load', setup);
-  document.addEventListener('cable-ready:after-morph', setup);
+  document.addEventListener('cable-ready:after-morph', event => {
+    setup();
+    if (event.detail.stimulusReflex) {
+      dispatch('stimulus-reflex:success', event.detail.stimulusReflex, stimulusApplication);
+      dispatch('stimulus-reflex:complete', event.detail.stimulusReflex, stimulusApplication);
+    }
+  });
+  document.addEventListener('stimulus-reflex:500', event => {
+    dispatch('stimulus-reflex:error', event.detail.stimulusReflex, stimulusApplication);
+    dispatch('stimulus-reflex:complete', event.detail.stimulusReflex, stimulusApplication);
+  });
+
+  // document.addEventListener('stimulus-reflex:before', event => {
```

Comment:
What’s the reason for including these comments? Would the be better as documentation or are they future additions?

Reply:
These are debug artifacts. I'll remove them prior to wrapping up the PR.